lfucg-exactions/client/js/components/PaymentForm.js

```diff
@@ -315,6 +342,9 @@ function mapDispatchToProps(dispatch, params) {
                                 account_name: account.response.account_name,
                                 lot_id: value_id,
                                 address_full: value_name,
+                                lot_exactions: lot_id.response.lot_exactions.current_exactions,
```

Comment:
These only update if there is an account.  Account is not a required field for lots.